state/azure/cosmosdb/cosmosdb.go

```diff
@@ -39,9 +39,6 @@ import (
 type StateStore struct {
 	state.DefaultBulkStore
 	client      *documentdb.DocumentDB
-	collection  *documentdb.Collection
```

Comment:
Thanks to the latest updates in the upstream SDK, we can now refer to collections in Cosmos DB via their ID and not just the "self link". So there's no need to save these in the object anymore. The function `c.findCollection()` returns the link, generated every time.